python/dolfinx/fem/assemble.py

```diff
@@ -50,7 +50,7 @@ def pack_constants(
     def _pack(form):
         if form is None:
             return None
-        elif isinstance(form, collections.abc.Iterable):
+        elif isinstance(form, collections.abc.Sequence):
```

Comment:
```suggestion
        elif isinstance(form, Sequence):
```
Maybe, also remove the `import collections`?